src/controllers/mainController.ts

```diff
@@ -499,11 +540,21 @@ export default class MainController implements vscode.Disposable {
     /**
      * Opens a new query and creates new connection
      */
-    public onNewQuery(): Promise<boolean> {
+    public async onNewQuery(sessionId?: string, content?: string): Promise<boolean> {
```

Comment:
>boolean [](start = 75, length = 7)

Do you want to update this to also return a URI since you did for newQuery()?

Reply:
Think it's fine since it doesn't need it